charts/bar-api/values.aat.template.yaml

```diff
@@ -5,3 +5,10 @@ java:
     S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://idam-api.aat.platform.hmcts.net
     PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
+  keyVaults:
```

Comment:
You can move them back to values.yaml ?Not sure why u want to do aat .. which means preview wont work.

Reply:
resolved

Reply:
resolved